docs/labwc-theme.5.scd

```diff
@@ -178,6 +182,12 @@ elements are not listed here, but are supported.
 	Menu title color. Default #589bda.
 	Note: A menu title is a separator with a label.
 
+*menu.title.bg.color*
+	Background color of separator label.
+
+*menu.title.text.color*
+	Text color of separator label.
```

Comment:
Add `Default #ffffff`

I know the Openbox man page says black, but we've generally been using the Clearlooks-3.4 colors, suggest we carry on with that.

And white goes better on the blue background too.